src/util.js

```diff
@@ -311,3 +311,16 @@ function setLocEnd(node, index) {
   }
 }
 util.setLocEnd = setLocEnd;
+
+// http://stackoverflow.com/a/7124052
+function htmlEscapeInsideDoubleQuote(str) {
+  return str
+    .replace(/&/g, '&amp;')
```

Comment:
The parser unescapes the string. So if you write `<div id="&amp;" />`, `str` here will be `&` and not `&amp;`